src/core/catchment/giuh/GiuhJsonReader.cpp

```diff
@@ -26,9 +26,23 @@ std::vector<double> GiuhJsonReader::extract_cumulative_frequency_ordinates(ptree
     std::vector<double> cumulative_freqs;
 
     // TODO: account for error condition of unmatching JSON structure for freqs
-    for (ptree::value_type freqs : catchment_data_node.get_child("CDF.CumulativeFreq")) {
+    std::string freq_node_name;
+    if (catchment_data_node.get_child_optional("CDF.CumulativeFreq") != boost::none) {
+        freq_node_name = "CDF.CumulativeFreq";
+    }
+    else {
+        freq_node_name = "CDF.minHydro.Runoff";
```

Comment:
I don't believe this key in the GIUH is a cumulative freq, I think it is a computed incremental run off.  My suspicion is that if this else case is ever encountered, the runtime_error below will always trigger.

Reply:
Just throw a runtime error here for now "GIUH doesn't contain CumlativeFreq key"

Reply:
Change applied.